### PR TITLE
Added the listing of available games to the homepage.

### PR DESCRIPTION
**Pull request by ghost**
-So if there is an open_seat available in a game it will be displayed on the homepage by the name it was given during creation.

-An open_seat is a nil value for the black_player_id or a white_player_id. So if a game doesn't have both of those set it will be displayed.

-Clicking the link will bring you to the show page for that individual game.

-I also fixed a bunch of the RuboCop styling errors that were in the Rspec tests.

-And I modified the rails version to be 4.0.6 so I would stop getting a circular time-out error. https://gyazo.com/82491e6dbb4f7cc1c5dc44d8b479597f

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class StaticPagesController < ApplicationController
   layout :layout_for_page
 
   def show
+    @games = Game.all
```
Comment (woodbrettm) [inline app/controllers/static_pages_controller.rb]: This should go in the games controller in def index. def index in the games controller is the home page for when the user is logged in. static_pages#show, id='home' is the home page before the user logs in :)

Comment (ghost) [inline app/controllers/static_pages_controller.rb]: Moved it, thanks I was a bit confused about the homepage.

File: app/views/games/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<h2>Available Games</h2>
+<hr  />
+
+<% @games.each do |game| %>
+  <% if game.open_seat? == true %>
```
Comment (mscoutermarsh) [inline app/views/games/index.html.erb]: this could also just be `game.open_seat?` since it returns true or false

Comment (mscoutermarsh) [inline app/views/games/index.html.erb]: `<% if game.open_seat? %>`

File: spec/models/game_spec.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  describe 'Game status' do
+    it 'should not have an open seat available' do
+      game = Game.create(name: 'test', white_player_id: 1, black_player_id: 2)
+      expect(game.open_seat?).to eq nil
```
Comment (mscoutermarsh) [inline spec/models/game_spec.rb]: I'd expect this to be `false`

**Pull request merged**